test/jdk/javax/swing/JMenu/TestMenuMnemonic.java

```diff
@@ -0,0 +1,133 @@
+    private static JMenuItem item1;
+    private static JMenuItem item2;
+    private static int expectedMnemonicShowHideCount = 5;
+
+    public static void main(String[] args) throws Exception {
+        UIManager.setLookAndFeel("com.sun.java.swing.plaf.windows.WindowsLookAndFeel");
+        Robot robot = new Robot();
+        robot.setAutoDelay(100);
+        int mnemonicHideCount = 0;
+        int mnemonicShowCount = 0;
+        try {
+            SwingUtilities.invokeAndWait(() -> {
+                createAndShowUI();
+            });
+
+            robot.waitForIdle();
+            robot.delay(1000);
+
+            for (int i = 0; i < 10; i++) {
+                robot.keyPress(KeyEvent.VK_F10);
+                robot.waitForIdle();
+                robot.delay(50);
+                robot.keyRelease(KeyEvent.VK_F10);
+                MenuSelectionManager msm =
+                        MenuSelectionManager.defaultManager();
+                MenuElement[] selectedPath = msm.getSelectedPath();
+                if (WindowsLookAndFeel.isMnemonicHidden()) {
+                    mnemonicHideCount++;
+                    // check if selection is cleared when mnemonics are hidden
+                    if (selectedPath.length != 0) {
+                        throw new RuntimeException("Menubar is active even after" +
+                                " mnemonics are hidden");
+                    }
+                } else {
+                    mnemonicShowCount++;
+                    if (selectedPath.length == 0 &&
+                        (selectedPath[0] != menuBar || selectedPath[1] != fileMenu)) {
+                        throw new RuntimeException("No Menu and Menubar is active when" +
+                                " mnemonics are shown");
+                    }
+                }
+            }
+            robot.waitForIdle();
+            robot.delay(1000);
+            if (mnemonicShowCount != expectedMnemonicShowHideCount
+                && mnemonicHideCount != expectedMnemonicShowHideCount) {
+                throw new RuntimeException("Mismatch in Mnemonic show/hide on F10 press");
+            }
+
+
+        } finally {
+            SwingUtilities.invokeAndWait(() -> {
+                if (frame != null) {
+                    frame.dispose();
+                }
+            });
+        }
+    }
+
+    private static void createAndShowUI() {
+        frame = new JFrame("Test Menu Mnemonic Show/Hide");
+        menuBar  = new JMenuBar();
+        fileMenu = new JMenu("File");
+        editMenu = new JMenu("Edit");
+        fileMenu.setMnemonic(KeyEvent.VK_F);
+        editMenu.setMnemonic(KeyEvent.VK_E);
+        item1 = new JMenuItem("Item 1");
+        item2 = new JMenuItem("Item 2");
+        fileMenu.add(item1);
+        fileMenu.add(item2);
+        menuBar.add(fileMenu);
+        menuBar.add(editMenu);
+        frame.setJMenuBar(menuBar);
+        frame.pack();
+        frame.setSize(250, 200);
```

Comment:
```suggestion
        frame.setSize(250, 200);
```
You set the size, `pack` is redundant.